### ISSUE TITLE
Disqus is not working

### ISSUE THREAD

**Issue by qifengzhou** [theme-next/hexo-theme-next#15]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
Disqus works after loading.

### Actual behavior 
Disqus does not show up at the end of blogs.

### Steps to reproduce the behavior 
1. next-thema -config.yml
disqus:
enable: true
shortname: qifengzhou
count: true
2. hexo clean
hexo g # generate with the new settings
hexo s # start it local
3. hexo d # deploy also not working
* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```
node -v
v8.9.4

npm -v
5.6.0

```

### Hexo Information

```
hexo-cli: 1.0.4
os: Darwin 14.5.0 darwin x64
http_parser: 2.7.0
node: 8.9.4
v8: 6.1.534.50
uv: 1.15.0
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.25.0
openssl: 1.0.2n
icu: 59.1
unicode: 9.0
cldr: 31.0.1
tz: 2017b
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.4.4"
  },
  "dependencies": {
    "hexo": "^3.2.0",
    "hexo-deployer-git": "^0.3.1",
    "hexo-generator-archive": "^0.1.4",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-index": "^0.2.0",
    "hexo-generator-tag": "^0.2.0",
    "hexo-renderer-ejs": "^0.3.0",
    "hexo-renderer-marked": "^0.3.0",
    "hexo-renderer-stylus": "^0.3.1",
    "hexo-server": "^0.2.0"
  }
}
```

### NexT Information

**NexT Version:**

```
[] Latest Master branch.
[] Latest Release version.
[] Old version -
```

**NexT Scheme:**

```
[] All schemes
[x] Muse
[] Mist
[] Pisces
[] Gemini
```

### Other Information 
chrome

**Comment by ivan-nginx** [comment]
So, what version of NexT u use? Try to reproduce in v.6.0.0 without any 3rd-party plugins.

**Comment by qifengzhou** [comment]
Tried several things with information from other issues:
/themes/next/layout/_third-party/comments/disqus.swig
1. add 'https://'
        before: this.page.url = ’{{ page.permalink }}';
        after: this.page.url = 'https://' + ’{{ page.permalink }}';
2. remove this.page.url = ’{{ page.permalink }}';

not working

**Comment by ivan-nginx** [comment]
HTTPS there is no problem, because disqus must work in localhost. There is screenshot from localhost, above.

**Comment by qifengzhou** [comment]
Thank you ivan-nginx.
I am not sure how to get the version information of NexT there?

**Comment by ivan-nginx** [comment]
https://qifengzhou.github.io - see in the bottom — u use v5.1.4

**Comment by qifengzhou** [comment]
NexT version updated to v 6.0.x
-config.yml updated as above
# Disqus
disqus:
  enable: true
  shortname: qifengzhou
  count: true
Disqus is still not working.

**Comment by ivan-nginx** [comment]
![image](https://user-images.githubusercontent.com/16944225/34639861-106c01de-f2f9-11e7-9776-39be2408fde7.png)
Please, check your [admin disqus dashboard](https://disqus.com/admin). Is there any comment?

**Comment by qifengzhou** [comment]
There is no comments yet.
<img width="718" alt="screen shot 2018-01-06 at 1 53 04 pm" src="https://user-images.githubusercontent.com/26200327/34639911-34593d38-f2e9-11e7-876a-c1a5c3a15219.png">

General seetings
<img width="698" alt="screen shot 2018-01-06 at 1 55 19 pm" src="https://user-images.githubusercontent.com/26200327/34639914-4be92594-f2e9-11e7-89cb-a43a99afec1d.png">
<img width="674" alt="screen shot 2018-01-06 at 1 55 58 pm" src="https://user-images.githubusercontent.com/26200327/34639919-599f6fb8-f2e9-11e7-817d-8603e512d14b.png">

**Comment by ivan-nginx** [comment]
Well, u need to set up «Website URL» to «https://qifengzhou.github.io».

**Comment by qifengzhou** [comment]
the website URL is set at the general setting of disqus site
<img width="732" alt="screen shot 2018-01-06 at 2 01 01 pm" src="https://user-images.githubusercontent.com/26200327/34639962-2abf9c1c-f2ea-11e7-8eda-eae61061ae95.png">
still not working.

**Comment by ivan-nginx** [comment]
![image](https://user-images.githubusercontent.com/16944225/34639983-512ce2f4-f2fb-11e7-949d-3bd81dffe4b3.png)
Try to regenerate/redeploy site.

**Comment by qifengzhou** [comment]
hexo clean
hexo g
hexo d

# still dont see disqus

**Comment by ivan-nginx** [comment]
@qifengzhou see instructions above.

**Comment by ivan-nginx** [comment]
@qifengzhou can u show your full Hexo root `_config.yml`?

**Comment by qifengzhou** [comment]
Hexo root _config.yml

# Hexo Configuration
## Docs: https://hexo.io/docs/configuration.html
## Source: https://github.com/hexojs/hexo/

# Site
title: Qifeng Zhou
subtitle:
description: My thought and stuff
author: Qifeng Zhou
language: en
timezone: 


# URL
## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
url: ‘http://qifengzhou.github.io'
root: /
permalink: :year/:month/:day/:title/
permalink_defaults:

# Directory
source_dir: source
public_dir: public
tag_dir: tags
archive_dir: archives
category_dir: categories
code_dir: downloads/code
i18n_dir: :lang
skip_render:

# Writing
new_post_name: :title.md # File name of new posts
default_layout: post
titlecase: false # Transform title into titlecase
external_link: true # Open external links in new tab
filename_case: 0
render_drafts: false
post_asset_folder: false
relative_link: false
future: true
highlight:
  enable: true
  line_number: true
  auto_detect: false
  tab_replace:
  
# Home page setting
# path: Root path for your blogs index page. (default = '')
# per_page: Posts displayed per page. (0 = disable pagination)
# order_by: Posts order. (Order by date descending by default)
index_generator:
  path: ''
  per_page: 10
  order_by: -date
  
# Category & Tag
default_category: uncategorized
category_map:
tag_map:

# Date / Time format
## Hexo uses Moment.js to parse and display date
## You can customize the date format as defined in
## http://momentjs.com/docs/#/displaying/format/
date_format: YYYY-MM-DD
time_format: HH:mm:ss

# Pagination
## Set per_page to 0 to disable pagination
per_page: 10
pagination_dir: page

# Extensions
## Plugins: https://hexo.io/plugins/
## Themes: https://hexo.io/themes/
theme: next-reloaded

# Deployment
## Docs: https://hexo.io/docs/deployment.html
deploy:
  type: git
  repo: https://github.com/qifengzhou/qifengzhou.github.io.git
  branch: master

**Comment by ivan-nginx** [comment]
```yml
# URL
## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
url: ‘http://qifengzhou.github.io'
```

See instructions above.

**Comment by qifengzhou** [comment]
Should I put the contents in the hexo root _config.yml?
I didnt get it here.

**Comment by ivan-nginx** [comment]
@qifengzhou i repeat:

> See instructions [above](#issuecomment-355745807).

**Comment by qifengzhou** [comment]
I see..
The url should be out of the quote.
Now it works.
Thank you @ivan-nginx

**Comment by ivan-nginx** [comment]
> out of the quote

And with **httpS**.

U are welcome.
